src/components/ModalAdd/ModalAdd.tsx

```diff
@@ -0,0 +1,54 @@
+import {useState} from 'react';
+import {useDispatch} from 'react-redux';
+import {modalAddLeadersActions} from 'redux/modal/modal-actions';
+import {toast} from 'react-toastify';
+import {ILeaderFirst} from '../../redux/leaders/interfaces/index';
+import {addLeaders} from '../../redux/leaders/leaders-actions';
+import styles from './ModalAdd.module.scss';
+
+const ModalAdd = () => {
+  const [leaderName, setLeaderName] = useState('');
+  const [leaderScore, setLeaderScore] = useState('_');
```

Comment:
in this case, it is better to set the initial value as null 
 const [leaderScore, setLeaderScore] = useState<number>(null);